qa/tasks/deepsea.py

```diff
@@ -22,6 +22,19 @@
 
 log = logging.getLogger(__name__)
 deepsea_ctx = {}
+default_log_anchor = "WWWW: "
+
+
+def log_error_with_anchor(logger, anchor_str, message):
```

Comment:
Why these methods are required, if you already have all those self._log**** methods, have can have only:
```
def _anchored(anchor, message):
  return anchor + message
```